dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/RumContext.kt

```diff
@@ -13,6 +13,7 @@ import java.util.UUID
 internal data class RumContext(
     val applicationId: String = NULL_UUID,
     val sessionId: String = NULL_UUID,
+    val isSessionActive: Boolean = true,
```

Comment:
I believe it should be `false` by default, to have a consistency with default `sessionId` which is `NULL_UUID`, meaning there is no session.